CHANGELOG.md

```diff
@@ -57,6 +57,7 @@
 - Bugfix: Fixed a bug on Wayland where tooltips would spawn as separate windows instead of behaving like tooltips. (#4998, #5040)
 - Bugfix: Fixes to section deletion in text input fields. (#5013)
 - Bugfix: Show user text input within watch streak notices. (#5029)
+- Bugfix: Fixed draggable popups falling back to a non-native drag implementation on Windows. (#5051)
```

Comment:
If this bugfix doesn't fix something from the last stable release, it should just add its number to the changelog that caused it